inc/html.class.php

```diff
@@ -1103,15 +1103,12 @@ static function includeHeader($title='') {
       self::header_nocache();
 
       // Start the page
-      echo "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\"
-             \"http://www.w3.org/TR/html4/loose.dtd\">";
-      echo "\n<html><head><title>GLPI - ".$title."</title>";
-      echo "<meta http-equiv='Content-Type' content='text/html; charset=utf-8'>";
+      echo "<!DOCTYPE html>\n";
+      echo "<html lang=\"{$CFG_GLPI["languages"][$_SESSION['glpilanguage']][3]}\">";
+      echo "<head><title>GLPI - ".$title."</title>";
+      echo "<meta charset=\"utf-8\">";
 
-      // Send extra expires header
-      echo "<meta http-equiv='Expires' content='Fri, Jun 12 1981 08:20:00 GMT'>\n";
-      echo "<meta http-equiv='Pragma' content='no-cache'>\n";
```

Comment:
HTML meta tags for caching are ignored; and apparently for a long time.

The good way to go is to set cache stuff in headers, and it is already done ;)